e2e/utils.go

```diff
@@ -321,10 +325,16 @@ func createRBDStorageClass(c kubernetes.Interface, f *framework.Framework, scOpt
 	}
 	sc.Namespace = cephCSINamespace
 
-	if scOptions["volumeBindingMode"] == "WaitForFirstConsumer" {
-		value := scv1.VolumeBindingWaitForFirstConsumer
-		sc.VolumeBindingMode = &value
+	for key, val := range scOptions {
+		if key == "WaitForFirstConsumer" {
+			value := scv1.VolumeBindingWaitForFirstConsumer
+			sc.VolumeBindingMode = &value
+		}
+		if key == "mountOptions" {
```

Comment:
probably should be `else if`, or maybe a `switch`, but I guess the linter would complain about it.